### ISSUE TITLE
Icons in Notification component have invalid aria-hidden value

### ISSUE THREAD

**Issue by joseph-parsley** [buefy/buefy#3318]
### Overview of the problem

**Buefy** version: 0.9.3
**Vuejs** version: 2.6.11
**OS/Browser**: Windows 10/Chrome

### Description

The `b-notification` component with a `has-icon` attribute renders an icon. The icon renders with an `aria-hidden` attribute with no value, which is incorrect; it should be `aria-hidden="true"`.

(I know I'm not using the latest version but I couldn't see anything mentioning `aria-hidden` in the changelog for the newer versions.)

### Steps to reproduce

Mount a `b-notification` component with the `has-icon` attribute. The example I'm using has these attributes in case any combination of them contributes to this error:

```
<b-notification type="is-info" has-icon :closable="false">
```

### Expected behavior

The icon should properly have `aria-hidden="true"`.

### Actual behavior

The icon has `aria-hidden` with no value.

**Comment by joseph-parsley** [comment]
A colleague pointed out to me that the `svg` element does properly have `aria-hidden="true"`, but the `span` element above it has the invalid value referenced above. So it probably works okay for screen readers but this triggers a Lighthouse scan false positive since that's an invalid value.
